src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

```suggestion
h1,
p {
  margin: 0;
}
```
Look, here you are repeating yourself. You write `margin: 0` both in your `h1` and `p` selectors. You may write as I've shown above and remove `margin: 0` from your `h1` selector

---

I see. I thought it would be worse if I repeat tag h1 twice. I'll do it right now. Thanks